languages/julia/concepts/iterator-protocol/links.json

```
@@ -0,0 +1,2 @@
+[
+]
```

```suggestion
[]
```

Otherwise the prettier will probably do it at some point.

---

it didn't complain but good call anyway :)